Dockerfile

```diff
@@ -1,22 +1,12 @@
-# build environment
-FROM node:16-alpine as build
+FROM node:16-alpine
 
 WORKDIR /app
-ENV PATH /app/node_modules/.bin:$PATH
 
-COPY package.json ./
-COPY package-lock.json ./
-COPY yarn.lock ./
+COPY package.json /app
```

Comment:
you can just use `.` for copy because you already set `WORKDIR`

Reply:
So, there's a concept of docker layer caching. If I copy the `package.json` and `yarn.lock` before and install the deps, it will be cached for all subsequent builds (unless the `package.json` and `yarn.lock` changes)